Android/Nothing/app/quality.gradle

```diff
@@ -0,0 +1,15 @@
+detekt {
+    toolVersion = "1.14.1"
+    config = files("$rootProject.projectDir/config/detekt/detekt.yml")
+
+    reports {
+        xml {
+            enabled = true
+//            destination = file("$project.reporting.baseDir/detekt.xml")
```

Comment:
Is this going to be used in a near future? if not then move it to alv

Reply:
Can be used on tools like bitrise, sonarqube or similar.